lib/vector-common/src/internal_event/events_sent.rs

```diff
@@ -10,25 +10,28 @@ pub struct EventsSent<'a> {
     pub count: usize,
     pub byte_size: usize,
     pub output: Option<&'a str>,
+    pub source: Option<&'a str>,
 }
 
 impl<'a> InternalEvent for EventsSent<'a> {
```

Comment:
Note that this source has changed recently and no longer contains this implementation. You'll need to update against current master.

Reply:
Thanks! Yeah, I did find that out during a rebase this morning. I've got some time scheduled tomorrow to rectify this.